### DISCUSSION TITLE
Random Unknown Person Shown as the committer to my private repo

### DISCUSSION THREAD

**Discussion by tmtech90** [community/community#39891] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

When I did 'git push origin main' from my AWS EC2 instance into my PRIVATE Repo (just created with only me working on it) it says my commit was done by some random person. This seems like a real person and I have no idea who they are. Maybe it's someone who was associated with my AWS IP address before? Is this a security concern?

**Comment by radar-blip** [comment, marked as answer]
I dont think its a security concern, most likely someone using the same email as yours.
There is a similar discussion here: #29903

The last answer there may or may not help.
Good luck!

**Reply by tmtech90** [reply]
Thanks. Yea I changed the email in git config and committed again and it showed my name this time. Very weird.

**Reply by radar-blip** [reply]
You're welcome. Happy to help. 
Glad you were able to fix it!
